src/Raven.Client/Documents/Operations/TimeSeries/GetTimeSeriesOperation.cs

```diff
@@ -104,9 +110,9 @@ public override void SetResponse(JsonOperationContext context, BlittableJsonRead
                 if (response == null)
                     return;
 
-                Result = JsonDeserializationClient.TimeSeriesRangeResult(response);
+                Result = JsonDeserializationBase.GenerateJsonDeserializationRoutine<TimeSeriesRangeResult<TValues>>()(response);
```

Comment:
Do we want to cache those routines? Now it generates them everytime?

Reply:
Can we cache those? Will try but not sure it's possible because of `TValues`.

Reply:
done